kitsune/sumo/static/sumo/scss/base/forms/_fields.scss

```diff
@@ -406,11 +414,34 @@ textarea {
 }
 
 .checkbox-list {
-  display: grid;
-  grid-auto-flow: row;
-  grid-template-columns: repeat(3, 1fr);
-  grid-template-rows: masonry;
-  gap: p.$spacing-sm p.$spacing-sm;
+  display: block;
+  margin: 0 0 p.$spacing-xl 0;
+  padding: 0;
+  list-style: none;
+
+  @media #{p.$mq-md} {
+    display: grid;
+    grid-auto-flow: row;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: masonry;
+    gap: p.$spacing-sm p.$spacing-sm;
+  }
+
+  &--sublist {
+    margin-left: 24px;
+    
+    @include p.bidi(((margin-left, 24px, 0),
+        (margin-right, 0, 24px),
+    ));
+
+    @media #{p.$mq-md} {
+      margin-left: 32px;
+      
+      @include p.bidi(((margin-left, 32px, 0),
+          (margin-right, 0, 32px),
```

Comment:
Can these `px` values be replaced with `p.$spacing-?` values?

Reply:
They can - got it!